examples/standard/http-redirect-rewrite/httproute-redirect-https.yaml

```
@@ -1,15 +1,18 @@
 #$ Used in:
 #$ - site-src/api-types/httproute.md
-apiVersion: gateway.networking.k8s.io/v1beta1
 kind: HTTPRoute
+apiVersion: gateway.networking.k8s.io/v1
```

Same comment as above
```suggestion
apiVersion: gateway.networking.k8s.io/v1
kind: HTTPRoute
```

---

Updated!